upstairs/src/lib.rs

```diff
@@ -1404,6 +1418,46 @@ impl Upstairs {
             }
         });
     }
+
+    /*
+     * Check the region information for a downstairs and decide if we should
+     * allow this downstairs to be added.
+     * TODO: Still a bunch to do here around reconnecting.
+     */
+    fn add_downstairs(
+        &self,
+        client_id: u8,
+        rdef: RegionDefinition,
+    ) -> Result<()> {
+        println!("[{}] Got region def {:?}", client_id, rdef);
+
+        let mut uuid_hm = self.ds_uuid.lock().unwrap();
+        if let Some(uuid) = uuid_hm.get(&client_id) {
+            if *uuid != rdef.uuid() {
+                panic!(
+                    "New client:{} uuid:{}  does not match existing {}",
+                    client_id,
+                    rdef.uuid(),
+                    uuid
+                );
+            } else {
+                println!(
+                    "Returning clientL{} UUID:{} matches",
```

Comment:
`client:`

Reply:
Ha,  I just saw that typo last night, and almost pushed an update.  Will fix.